src/legacy/List/Filters.component.js

```diff
@@ -231,16 +233,19 @@ export default class Filters extends React.Component {
                                     checked={showOnlyActiveUsers}
                                 />
                             </Grid>
-                            <Grid item xs={6} className="control-row switch">
+                            <Grid item xs={4} className="control-row switch">
                                 <span>{this.getTranslation("Filtering_behavior")}</span>
                                 <div className="control-switch">
-                                    <span>{this.getTranslation("AND")}</span>
+                                    <span>{this.getTranslation("OR")}</span>
                                     <Switch
                                         className="control-switch"
-                                        onChange={this.setFilter("rootJunction", this.checkboxHandler)}
-                                        checked={rootJunction}
+                                        onChange={() => {
+                                            const newRootJunction = rootJunction === "AND" ? "OR" : "AND";
+                                            this.setState({ rootJunction: newRootJunction }, this.notifyParent);
+                                        }}
+                                        checked={rootJunction === "AND"}
                                     />
-                                    <span>{this.getTranslation("OR")}</span>
+                                    <span>{this.getTranslation("AND")}</span>
```

Comment:
I am not familiar with the infrastructure of this project, but I see that we have a `public/old-i18n` and the gettext infrastructure is there. Maybe we should be using then `i18n.t` on new code?

Reply:
I wasn't familiar with the distinction of `old-i18n` vs `i18n.t`. The rest of the component uses `this.getTranslation`, and I'm happy to see how to change it, but I'm not sure this PR would be the place? What would you prefer?

Reply:
I see, we can leave this.getTranslation for existing components and i18n.t for the new ones.